examples/compiler/00_index.ipynb

```diff
@@ -0,0 +1,53 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Devito's Operator \n",
+    "\n",
+    "In Devito, an `Operator` carries out three fundamental tasks: generation of low-level code, JIT compilation, and execution. Overall, the lowering process performed by an `Operator` – from high-level equations to dynamically compiled and executable code – consists of multiple compiler passes, summarized in the Figure bellow. \n",
+    "\n",
+    "![Compiler Stages](https://i.ibb.co/dKjsZ72/compiler-stages.png)\n",
+    "\n",
+    "The following series of notebooks will expose key aspects of those compilation stages in a hands-on fashion. We will be exploring the heart of the `Operator` class, which can be found at `devito/operator.py`.\n",
```

Comment:
> will expose key aspects of those compilation stages in a hands-on fashion

I'd rephrase it as

"will describe these compilation passes in a hands-on fashion"

Then:

"We will explore the heart .."  ("will be exploring" here sounds odd)

Then another thing:

"of the `Operator` class, ..."

two comments here:

1) I'd write "of the `Operator` class, which orchestrates code generation and compilation."
2) I'd add a hover link to devito/operator.py -- I think the syntax is \[Operator class\](https://github.com/opesci/devito/blob/master/devito/operator.py)"

Reply:
fixed